introduction.md

```diff
@@ -0,0 +1,2 @@
+"Hello sirs, this is my introduction..greatness and beauty combined"
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.